lib/flowViewport/sceneIndex/fvpPruneTexturesSceneIndex.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright 2024 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "flowViewport/sceneIndex/fvpPruneTexturesSceneIndex.h"
+#include "pxr/base/tf/staticTokens.h"
```

Comment:
Nitpick : I think these should have the #include <pxr/base/tf/staticTokens.h> instead of the double quote.